endpoints/lib/vboxwebService-5.1.wsdl

```diff
@@ -5,7 +5,7 @@
   Generator: src/VBox/Main/webservice/websrv-wsdl-service.xsl
 -->
 <definitions xmlns:interface="urn:vbox" xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:vbox="http://www.virtualbox.org/" xmlns="http://schemas.xmlsoap.org/wsdl/" name="VirtualBox" targetNamespace="http://www.virtualbox.org/Service">
-  <import location="vboxweb-5.0.wsdl" namespace="http://www.virtualbox.org/"/>
+  <import location="vboxweb.wsdl" namespace="http://www.virtualbox.org/"/>
```

Comment:
I assume you either made a typo here and it should have be changed to -5.1 or you have a symlink since the file referenced here is currently in the repo and your PR does not create it.  Just thought I'd mention it.